backend/bitboxbase/bitboxbase.go

```diff
@@ -139,14 +121,14 @@ func (base *BitBoxBase) ConnectElectrum() error {
 	return nil
 }
 
-// GetUpdaterInstance return ths current instance of the updater
-func (base *BitBoxBase) GetUpdaterInstance() *updater.Updater {
-	return base.updaterInstance
+//GetRPCClientInstance returns ths current instance of the rpcClient
+func (base *BitBoxBase) GetRPCClientInstance() *rpcclient.RPCClient {
```

Comment:
in the rest of the codebase we usually don't have the `Get` for getters, e.g. just `RPCClientInstance()` is fine. I'd even prefer `RPCClient()`, as obviously it is an instance ;)